src/components/UserInfo.jsx

```diff
@@ -1,28 +1,27 @@
 import '../reset.css';
 import './UserInfo.scss';
+import React, { useEffect, useState } from 'react';
+import axios from 'axios';
 
 function UserInfo() {
+  const [users, setUsers] = useState([]);
```

Comment:
You fetch only *one* user -> `[user, setUser]`

Reply:
okay, no problem i will replace this immediately